### ISSUE TITLE
[QUESTION]  Pass numpy slice from Python to C++

### ISSUE THREAD

**Issue by YuraCobain** [pybind/pybind11#2965]
Is it possible to pass numpy slice from Python to C++?
I couldn't find any example from Python to C++.

**Comment by cjaques** [comment]
Yes, just create a wrapper accepting a py::array_t and, on the python side, call it with a sliced array.

**Comment by YuraCobain** [comment]
Thank you for your answer.
But still it's unclear how to work with it.
Due to it's not continues memory region I need the base address of original array and offsets from it to the start of slice to interpret it correctly. 
Array is slice but not vise versa.
